<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="start_btn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_sq">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>sequencing_method</string>
   </property>
   <widget class="QRadioButton" name="groupBox_m_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>method1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="groupBox_m_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>method2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="groupBox_m_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>method3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="groupBox_m_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>method4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="groupBox_m_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>method5</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
